src/html/index.html

```diff
@@ -47,13 +52,13 @@
     <link rel="stylesheet" href="{{{ cdn }}}/css/fonts.css" type="text/css" />
     <link rel="stylesheet" href="{{{ cdn }}}/css/styles.css" type="text/css" />
     <link rel="icon" href="{{{ cdn }}}/images/favicon.png" type="image/png" sizes="196x196" />
-  </head> 
+  </head>
   <body data-decorator="load-ga display-rp-name prevent-clickjack">
-    <!--
-    <div class="banner">
-      <p></p>
+    <div data-decorator="check-if-maintenance-mode" hidden>
+      <div class="banner">
+        <p>We are currently experiencing an issue &amp; working hard to resolve it. Please try to login again in a few minutes.  Request help via <a href="mailto:moc@mozilla.com">moc@mozilla.com</a>, IRC #moc or Slack #mozsm-all.</p>
```

Comment:
another nit i forgot :(
`IRC <a href="https://mozilla.slack.com">#mozsm-all</a>` (or similar if you know how to reliably direct link the chanel) and `IRC <a href="irc://irc.mozilla.org/moc">#moc</a>`